scripts/run-parachain.ts

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/env -S deno run -A --unstable
+
+/// This a script to run and setup the dkg network parachain.
+
+import {
+  ApiPromise,
+  Keyring,
+  WsProvider,
+} from "https://esm.sh/@polkadot/api@9.2.3?bundle";
+import { Input } from "https://deno.land/x/cliffy@v0.24.3/prompt/mod.ts";
+import { assert } from "https://deno.land/std@0.152.0/testing/asserts.ts";
+import {
+  keypress,
+  KeyPressEvent,
+} from "https://deno.land/x/cliffy@v0.24.3/keypress/mod.ts";
+
+type ScrtipState = {
```

Comment:
What is this supposed to be @shekohex ? Not sure I understand the naming.

Reply:
Oh, I will fix that, it should be `ScriptState`.